homeassistant/components/shelly/binary_sensor.py

```diff
@@ -281,8 +281,8 @@ def __init__(
     "mute": RpcBinarySensorDescription(
         key="flood",
         sub_key="mute",
-        name="Mute",
-        device_class=BinarySensorDeviceClass.SOUND,
+        value=lambda status, _: not status["mute"],
+        name="Alarm sound",
```

Comment:
Why not calling this "Mute"? (not the device class, the sensor name), it is mute alarm button.

Reply:
I changed it based on @bieniu request. I prefer "Mute" as well but....

Reply:
As far as I understood he asked to change the device class, not the name

Reply:
He wrote: `My proposition: Alarm sound and states On/Off.`

Reply:
I will try to resolve this with him 👍

Reply:
`Mute` with states `On`/`Off` is fine form me.